src/PROPOSAL/PROPOSAL/Propagator.h

```diff
@@ -33,9 +33,8 @@ class Propagator {
 private:
     Interaction::Loss DoStochasticInteraction(
         ParticleState&, PropagationUtility&, std::function<double()>);
-    int AdvanceParticle(ParticleState& p_cond, const double E_f,
-                        const double max_distance,
-        std::function<double()> rnd, Sector& current_sector);
+    int AdvanceParticle(ParticleState&, const double, const double,
```

Comment:
Why are there no parameter names anymore?

Reply:
No real reason. They are not needed in the header file. But I could put them back for convenience.